RadixWallet/Features/TransactionReviewFeature/TransactionReview+View.swift

```diff
@@ -495,6 +509,7 @@ struct ExpandableTransactionHeading: View {
 				Spacer(minLength: 0)
 			}
 		}
+		.padding(.trailing, .huge2) // padding from the vertical dotted line
```

Comment:
To be clear, this is not the padding counting from the line, right? It's from the end of the view?

Reply:
yes, from the end of the view. I should have better a more descriptive constant constant